plugins/contrib/jobs/build/use.yaml

```diff
@@ -28,29 +27,57 @@ runs:
     volumes:
       - name: buildkitd
         emptyDir: {}
+      - name: certs
+        secret:
+          secretName: "{{ or $.with.buildkitServiceClientCertsSecretName `buildkit-client-certs` }}"
+          optional: true
```

Comment:
necessary ?

Reply:
yep mTLS is very recommended, we should use it IMHO, but even if we decide to not implement it, this will not be blocking (the secret is optional) and we could implement it easier if an audit require it